src/g_canvas.c

```diff
@@ -801,13 +801,14 @@ static void canvas_drawlines(t_canvas *x)
 {
     t_linetraverser t;
     t_outconnect *oc;
+    int iyoffset = (IHEIGHT/2)*x->gl_zoom, oyoffset = (OHEIGHT/2)*x->gl_zoom;
```

Comment:
this seems to do integer arithmetic, potentially getting an unintended 1-pixel offset.
what about the following (untested)?

~~~C
int iyoffset = IHEIGHT*0.5*x->gl_zoom, oyoffset = OHEIGHT*0.5*x->gl_zoom;
~~~